addons/anonymization/models/anonymization.py

```diff
@@ -64,15 +64,16 @@ def _get_model_and_field_ids(self, vals):
             return (field.model_id.id, field.id)
         return (False, False)
 
-    @api.model
-    def create(self, vals):
+    @api.create_multi
+    def create(self, valses):
```

Comment:
Why not `*valses`? It would do the same, but it would keep most backwards compatibility. :thinking: Otherwise we have to change every single call to `.create()` in every addon.

Reply:
Also, better variable name is values instead of valses.

Reply:
@Yajo I will try using multiple arguments instead of a list. I have thought about that API, but discarded for some reason I can't remember. I will tell you when memory comes back :wink:

Anyway, we **don't have to** change every single definition of `create` thanks to the decorator.

Reply:
@Yenthe666 yeah, naming things is difficult. The name `values` is already used everywhere for **one** dictionary of field values :confused: That is also where `vals` comes from. The name `valses` is a plural form of `vals` (considered as singular because it represents a dictionary). I am open to suggestions for a better name.

Reply:
@Yajo now I remember why `*args` is problematic.

Internally there is no problem, since `create` always returns a recordset. But the external API (xmlrpc, jsonrpc) of `create` becomes ambiguous: `create(vals)` used to return a single record id. Now it should return a list of ids. Backward compatibility with the old API is not possible, unless you make it more complex (return one id or a list depending on the number of arguments).

Using a list of dicts makes a clear difference between the "single" and "multi" API of create.

Reply:
Or may be we can keep variable name vals as it is and check If it is Dict or List in the end.

Reply:
`multivals`?

Reply:
or `vals_list`